aten/src/ATen/core/DistributionsHelper.h

```diff
@@ -35,21 +36,6 @@
 
 namespace at {
 
-// Using VectorType in Box-muller derived distributions to avoid
```

Comment:
Code motion only?

Reply:
Looks like this was deleted entirely

Reply:
yeah, deleted `VectorType` because it was unused, any objections?

Reply:
Nope, but since there's a lot of code that is moving around, I have to manually figure out what is being deleted and what is just being moved. That makes review harder.